frontend/__tests__/Pages/Inspection/Inspection.test.jsx

```diff
@@ -112,7 +112,7 @@ describe('Page displays bottom-level datasets from municipalities', () => {
   });
 
 
-  it('Shows the title and description of a single dataset', async () => {
+  it('Shows the title of a single dataset', async () => {
```

Comment:
Why remove description from the test?

Reply:
My bad, thought I needed to change this because of changes. The description was not correctly implemented for metadata.